src/users/components/UserList.vue

```diff
@@ -1,10 +1,21 @@
 <template>
   <div class="list-wrapper">
     <QList>
+      <QItem
+        v-if="users.length > 1"
+        v-t="{ path: 'GROUP.MEMBERLIST', args: { total: users.length } }"
```

Comment:
Did you intend to use `users` instead of `activeUsers`?

Reply:
Yep, that was intentional. Before I had both mentioned but that created a phrase that was too long. And sure, I could just use active ones, but I thought it could make sense to show the total amount, since inactive users get kicked out at some point anyways.